packages/react-scheduler/src/__tests__/SchedulingUMDBundle-test.js

```diff
@@ -16,25 +16,26 @@ describe('Scheduling UMD bundle', () => {
     jest.resetModules();
   });
 
+  function compareAPIS(apis) {
+    apis = apis.map(api => Object.keys(api).sort());
+    for (let i = 1; i < apis.length; i++) {
+      expect(apis[0]).toEqual(apis[i]);
```

Comment:
Might be a bit clearer if the "source of truth" API is passed as first argument. Then you can remove the `let i = 1` special case. Also errors would be better if you do

```js
expect(apis[i]).toEqual(nodeAPI);
```

because it's UMD/Secrets API that are *actual* and the original source code that is *expected*.